eng/Signing.props

```diff
@@ -33,6 +33,7 @@
 
   <ItemGroup>
     <!-- We don't produce font files. We rebundle some for using the web brower, so they do not need to be signed. -->
```

Comment:
Might be worth expanding this comment to include .js files

Reply:
To confirm, this file is shipping as a part of tye right now. Is that okay?

Reply:
I don't know of a reason why it wouldn't be